### DISCUSSION TITLE
How are these exploits found?

### DISCUSSION THREAD

**Discussion by nullkid10** [3kh0/ext-remover#1265] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I'd like to work on discovering exploits for new cros versions, and learn why old exploits don't work. If I could speak to someone who knows a bit more about this, i'd like to develop some things, or just pentest a bit. Thanks!

**Comment by S-PScripts** [comment]
testing trial and error and discoveries and information found that can be exploited in chromeos

updates patch the old exploits

join titanium network discord.gg/unblock for lots of people who make these exploits and shtuff
